cmake/FindIgnOGRE2.cmake

```diff
@@ -391,16 +391,25 @@ else() #WIN32
       list(APPEND OGRE2_INC_PATHS "${_rootPath}/include/${OGRE2_SEARCH_VER}")
   endforeach()
 
+  if(DEFINED ENV{CONDA_PREFIX})
+    list(APPEND OGRE2_PATHS $ENV{CONDA_PREFIX})
+    set(OGREMAIN_LIB_NAME "OgreNextMain")
+    set(OGRENEXT_PATH_SUFFIXES "Library/include/OGRE-Next")
```

Comment:
This is a bit brittle, as the presence or `Library` or not depends on weather Windows is used or not. Furthermore, ${CONDA_PREFIX}/Library should already be in the `CMAKE_SYSTEM_PREFIX_PATH`, so it should be just sufficient to add `OGRE-Next` to the `PATH_SUFFIXES`, even unconditionally.  Similarly, there should be no need for passing `$ENV{CONDA_PREFIX}` to `OGRE2_PATHS` , as the `${CONDA_PREFIX}/Library` is already in `CMAKE_SYSTEM_PREFIX_PATH` . Which cmake version are you using in the CI?